### PR TITLE
Allow edit external service email attributes

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

External service for sending email was missing a couple of attributes to be edited

## Screenshots, if any

Screenshot 2022-12-15 at 5 51 09

Screenshot 2022-12-15 at 8 21 59

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/system/external-services/email/edit-email/edit-email.component.html [hunk 1]
```diff
@@ -10,15 +10,20 @@
 
           <mat-form-field>
             <mat-label>Username</mat-label>
-            <input matInput required formControlName="username">
+            <input matInput type="email" required formControlName="username">
```
Comment (adamsaghy) [inline src/app/system/external-services/email/edit-email/edit-email.component.html:13]: username is not necessarily an email, so i dont think the type should be `email`

File: src/app/system/external-services/email/edit-email/edit-email.component.html [hunk 2]
```diff
@@ -41,20 +46,40 @@
           </mat-form-field>
 
           <mat-form-field>
+            <mat-checkbox *ngIf="emailConfigurationForm.value.legalFormId === 1" fxFlex="48%" labelPosition="before"
```
Comment (adamsaghy) [inline src/app/system/external-services/email/edit-email/edit-email.component.html:49]: legalform?

File: src/app/system/external-services/email/edit-email/edit-email.component.html [hunk 3]
```diff
@@ -41,20 +46,40 @@
...
+              formControlName="isStaff" class="margin-v">
```
Comment (adamsaghy) [inline src/app/system/external-services/email/edit-email/edit-email.component.html:50]: isStaff?

File: src/app/system/external-services/email/edit-email/edit-email.component.html [hunk 4]
```diff
@@ -41,20 +46,40 @@
...
             <input matInput required formControlName="useTLS">
             <mat-error *ngIf="emailConfigurationForm.controls.useTLS.hasError('required')">
               Use TLS is <strong>required</strong>
             </mat-error>
           </mat-form-field>
 
+          <mat-form-field>
+            <mat-label>From Email</mat-label>
+            <input matInput type="email" required formControlName="fromEmail">
+            <mat-error *ngIf="emailConfigurationForm.controls.fromEmail.hasError('required')">
```
Comment (adamsaghy) [inline src/app/system/external-services/email/edit-email/edit-email.component.html:63]: Can you help me to understand where this "from email" is coming from?

**Review by adamsaghy — CHANGES_REQUESTED** [review]
See my questions

File: src/app/system/external-services/email/edit-email/edit-email.component.html (hunk 3 above)
Comment (josehernandezfintecheandomx) [reply src/app/system/external-services/email/edit-email/edit-email.component.html]: Done updated!

File: src/app/system/external-services/email/edit-email/edit-email.component.html (hunk 2 above)
Comment (josehernandezfintecheandomx) [reply src/app/system/external-services/email/edit-email/edit-email.component.html]: Done updated!

File: src/app/system/external-services/email/edit-email/edit-email.component.html (hunk 1 above)
Comment (josehernandezfintecheandomx) [reply src/app/system/external-services/email/edit-email/edit-email.component.html]: Done removed!

**Review by adamsaghy — APPROVED** [review]
LGTM

**Pull request merged**